Meetings/devel-call-2025-05-21.md

```diff
@@ -0,0 +1,68 @@
+# Meeting Minutes: SVSM Development Call (May 21st, 2025)
+
+## Topics:
+
+### OpenSSL 3.5 Integration:
+
+* Oliver Steffen reported that his PR for OpenSSL 3.4.1 integration has been merged.
+* His next goal is to integrate OpenSSL 3.5, but this requires fixes in OpenSSL for firmware environments (which are already merged but not yet released)
+* A new release of the TPM library due to enforced version numbers is also needed.
+* Jon Lange asked about potential issues with not yet running on 3.5, to which Oliver replied he wasn't aware of any.
+* OpenSSL Performance Concerns: James Bottomley brought up recent reports about OpenSSL 3.5 performance problems.
```

Comment:
IIUC from https://lwn.net/Articles/1020309/ it was more a concern about OpenSSL 3, but reading the comments performance improved a bit e.g. from 3.0 to 3.2, but still slower than 1.1